splice_machine/pom.xml

```diff
@@ -23,6 +23,20 @@
         <version>3.0.0.3-SNAPSHOT</version>
     </parent>
     <dependencies>
+    	 
```

Comment:
the splice_machine module is specifically designed to NOT have this dependency. Any code which required hadoop or hbase should be moved to the hbase_sql module.